### DISCUSSION TITLE
Run kubelet with read only port 10255

### DISCUSSION THREAD

**Discussion by cloudcafetech** [rancher/rke2#1987] [Q&A] [answered]
Seems by default RKE2 not support to run kubelet with read only port 10255.

Can you please let me know how I can run Run kubelet with read only port 10255.

**Comment by brandond** [comment, marked as answer]
`--kubelet-arg=read-only-port=10255`

https://docs.rke2.io/install/install_options/linux_agent_config/

**Reply by cloudcafetech** [reply]
Thanks for quick response, will update you.

**Reply by cloudcafetech** [reply]
in config file I think need to add as below ..

```kubelet-arg: "read-only-port=10255"```

**Reply by brandond** [reply]
yes, that would be the config file version of the flag. Or, if you want to pass multiple args to the kubelet:
```yaml
kubelet-arg: 
 - "read-only-port=10255"
```

**Reply by cloudcafetech** [reply]
👍
